databuilder/contracts/base.py

```diff
@@ -103,3 +105,18 @@ def validate_schema(cls, schema):
             assert (
                 column.type.python_type == schema[col_name]
             ), f"{col_name} doesn't match {column}"
+
+
+def get_all_contracts():
```

Comment:
The only difference between this and `get_all_backends` is the class it's looking for subclasses on.  Is it worth making it a shared function?